script.js

```diff
@@ -311,7 +310,37 @@ function addLogout() {
     });
 }
 
+function addJumboCards() {
```

Comment:
this will look in the constants file to fill out the big cards at the top. you can take this approach if you want to also control other parts of the site with constants